lib/Net/Easypost/Address.pm

```diff
@@ -162,14 +174,19 @@ sub merge {
 =method verify
 
 This method takes a L<Net::Easypost::Address> object and verifies its underlying
-address
+address.
+
+If a non-US address is asked for verification, a warning will be
+emitted and the object itself will be returned.
 
 =cut
 
 sub verify {
     my $self = shift;
-    use Data::Dumper;
-
+    if ($self->country ne 'US') {
```

Comment:
Hunter McMillen <notifications@github.com> writes:

> In lib/Net/Easypost/Address.pm:
>
>>  
>>  =cut
>>  
>>  sub verify {
>>      my $self = shift;
>> -    use Data::Dumper;
>> -
>> +    if ($self->country ne 'US') {
>
> I'm confused, this still doesn't appear to support international
> shipping. Now it just explicitly states it is not supported. Is that
> what you intended? 

In my testing, I get international shipping, with this patch, with
international services (just look at the output of the test file).

From what I can see, it's just the verification of the address which is
missing for addresses located out of US. So I gues the ->verify thing
should be skipped if the address is not in US. Other things (like
getting rates) appear to work.

https://www.easypost.com/docs/api#addresses
"Currently only US based addresses can be verified."

Or am I missing something?

-- 
Marco